src/plugins/data_source/server/plugin.ts

```diff
@@ -45,6 +56,14 @@ export class DataSourcePlugin implements Plugin<DataSourcePluginSetup, DataSourc
       credentialSavedObjectsClientWrapper.wrapperFactory
     );
 
+    const dataSourceService: DataSourceServiceSetup = await this.dataSourceService.setup();
+
+    // Register plugin context to route handler context
+    core.http.registerRouteHandlerContext(
+      'data_source',
```

Comment:
`dataSource` instead of `data_source`? please check the convention of other context fields

Reply:
yes it should be camel case